### PR TITLE
Doc/update readme

### PR DESCRIPTION
**Pull request by a-bougouin**
### Purpose/goal of Pull Request
Update README to include mandatory setting `supported_langs`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,17 +75,44 @@ WOVN.io Ruby Library's valid parameters are as follows.
...
+	https://wovn.io/en/contact
+
+The library will redirect to the following URL.
+
+	https://wovn.io/contact
+
+### 2.3. supported_langs
+This tells the library which languages are being used on the website (including
+the original language). This setting allows for inserting metadata necessary to
+SEO (Search Engine Optimization).
```
Comment (tim-wovn) [inline README.md]: ...necessary for...

File: README.md [hunk 2]
```diff
@@ -69,17 +75,44 @@ WOVN.io Ruby Library's valid parameters are as follows.
...
+If your website is in English and you are using WOVN.io to localize it in
+Japanese, then you should use the following setting:
+```
+:supported_langs => ['en', 'ja']
+```
+**Note:** The order of the languages in the list does not matter.
+
+### 2.4. url_pattern
 
 The Library works in the Ruby Application by adding new URL's to be translated. You can set the type of url with the url_pattern parameter. There are 3 types that can be set.
```
Comment (tim-wovn) [inline README.md]: ...by adding new URLs to...

File: README.md [hunk 3]
```diff
@@ -107,25 +140,11 @@ If the default_lang is 'en', and the query is set to 'os', the above URL will be
...
-	https://wovn.io/en/contact
-
-The library will redirect to the following URL.
-
-	https://wovn.io/contact
-
-### 2.5 ignore_class
+### 2.6. ignore_class
 
 This sets "Ignore class" which prevent WOVN translating HTML elements that have one of the array.
```
Comment (tim-wovn) [inline README.md]: ...which prevents WOVN from translating... (2 corrections)

Also "one of the array" is incomplete.  This sentence can't be interpreted as is.

File: README.md [hunk 4]
```diff
@@ -69,17 +75,44 @@ WOVN.io Ruby Library's valid parameters are as follows.
...
 ### 2.1. project_token
 
 Set your WOVN.io Account's Project token. This parameter is required.
 
-### 2.2. url_pattern
+### 2.2. default_lang
+
+This sets the Ruby application's default language. The default value is English ('en').
+
+If a requested page, where the default language's parameter is included in the URL, the request is redirected before translating. The default_lang parameter is used for this purpose.
```
[the remark this answers is not in the document]
Comment (a-bougouin) [reply README.md]: done, thanks

File: README.md (hunk 1 above)
Comment (a-bougouin) [reply README.md]: done, thanks

File: README.md [hunk 5]
```diff
@@ -69,17 +75,44 @@ WOVN.io Ruby Library's valid parameters are as follows.
...
+
+If the default_lang is set to 'en', when receiving a request for the following URL,
+
+	https://wovn.io/en/contact
+
+The library will redirect to the following URL.
```
[the remark this answers is not in the document]
Comment (a-bougouin) [reply README.md]: done, thanks

File: README.md (hunk 2 above)
Comment (a-bougouin) [reply README.md]: done, thanks

File: README.md (hunk 3 above)
Comment (a-bougouin) [reply README.md]: done, thanks.

**Pull request merged**